src/modules/shared/services/toast-service/toast.service.tsx

```diff
@@ -0,0 +1,37 @@
+import { i18n } from 'next-i18next';
+import { toast, ToastOptions } from 'react-toastify';
+
+import { Toast, ToastProps } from '@shared/components';
+import { Optional } from '@shared/types';
+
+class ToastService {
+	public notify(
+		{
+			buttonLabel = i18n?.t('Ok') ?? '',
```

Comment:
Gaat de `Ok`-button altijd zichtbaar zijn? Imo voorzien we best dat die ook optioneel kan zijn

Reply:
Dat is de enige manier om de toast manueel te sluiten dus ja deze gaat altijd zichtbaar zijn, moest er een use case zijn waar dit niet nodig gaat zijn kunnen we het achteraf aanpassen.